xml/src/main/java/org/ehcache/xml/XmlConfiguration.java

```diff
@@ -83,16 +83,12 @@
 
 /**
  * Exposes {@link org.ehcache.config.Configuration} and {@link CacheConfigurationBuilder} expressed
- * in a XML file that obeys the ehcache-core.xsd (todo link this to proper location, wherever this ends up being)
+ * in a XML file that obeys the ehcache-core.xsd. (todo link this to proper location, wherever this ends up being)
```

Comment:
@chrisdennis is creating an issue.  Should this TODO be removed?  Replaced with a link _presuming_ the XSD is located at http://www.ehcache.org.v3?